include/circt/Dialect/Moore/MoorePasses.td

```diff
@@ -0,0 +1,27 @@
+//===--- Passes.td - Moore pass definition file ------------*- tablegen -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines the passes that work on the Moore dialect.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef CIRCT_DIALECT_MOORE_MOOREPASSES_TD
+#define CIRCT_DIALECT_MOORE_MOOREPASSES_TD
+
+include "mlir/Pass/PassBase.td"
+
+def DeleteLocalVar : Pass<"delete-local-var", "mlir::ModuleOp"> {
```

Comment:
Can we make this to SVModuleOp pass? It should automatically make the pass run in parallel.

Reply:
> Can we make this to SVModuleOp pass? It should automatically make the pass run in parallel.

Could you explain it in more detail? I want to learn more. Thanks in advance!
And what is the difference between `std::unique_ptr<mlir::Pass>` and `std::unique_ptr<OperationPass<ModuleOp>>`?

Reply:
Yes, the latter is more restrictive and pass manager only accepts ModuleOp. We can define the pass as SVModuleOp and we can nest the pass pipeline.

Reply:
Oh! I know what the means of `only accept ModuleOp`. Thanks!